graphql-dgs/src/main/kotlin/com/netflix/graphql/dgs/internal/DgsSchemaProvider.kt

```diff
@@ -188,13 +184,13 @@ class DgsSchemaProvider(
             mergedRegistry = mergedRegistry.merge(existingTypeDefinitionRegistry.get())
         }
 
-        val codeRegistryBuilder = GraphQLCodeRegistry.newCodeRegistry().fieldVisibility(fieldVisibility)
+        val codeRegistryBuilder = GraphQLCodeRegistry.newCodeRegistry()
         if (defaultDataFetcherFactory.isPresent) {
             codeRegistryBuilder.defaultDataFetcher(defaultDataFetcherFactory.get())
         }
 
         val runtimeWiringBuilder =
-            RuntimeWiring.newRuntimeWiring().codeRegistry(codeRegistryBuilder).fieldVisibility(fieldVisibility)
```

Comment:
I'm surprised this doesn't break disabling introspection for the non-spring-graphql version of the framework. Do we have a test for it?

Reply:
Yes, there is a test for it [here on the non-spring integration](https://github.com/Netflix/dgs-framework/blob/688a5db91bb0aabaf4f2c2c908b63acb767d6ee5/graphql-dgs-spring-boot-oss-autoconfigure/src/test/kotlin/com/netflix/graphql/dgs/autoconfig/DisabledIntrospectionTest.kt#L4) and I added one also on the [dgs-spring integration here](https://github.com/Netflix/dgs-framework/pull/1956/files#diff-11612275406ba1239d9ded889d1d58eae29ab7b27aeaf275108b5ecd36f660b0R241). 

What this part change does is refactor disabling introspection via the current deprecated way (configuring it on the executable schema via the RuntimeWiring) to the using the recommended way Instrospection.enabledJvmWide.

By default introspection is enabled so it doesn't need to be explicitly set, my reasoning was that the use case for setting this in the builder only applied when disabling introspection via the deprecated way but that's no longer the [best](https://github.com/graphql-java/graphql-java/blob/master/src/main/java/graphql/schema/visibility/NoIntrospectionGraphqlFieldVisibility.java#L16) way, this is why I removed it when refactoring to use Instrospection.enabledJvmWide

Reply:
I mentioned in another comment, but `GraphqlFieldVisibility` itself is not deprecated. I think we're taking away functionality here, since someone could be registering a custom implementation with their own logic.

Reply:
Good point, updated to not remove this functionality.